external/slowparse/slowparse.js

```diff
@@ -1382,6 +1392,12 @@
     },
 
     // This is a helper function to determine whether a given string
+    // is an obsolete HTML element tag
+    _knownObsoleteHTMLElement: function(tagName) {
+      return this.obsoleteHtmlElements.indexOf(tagName) > -1;
```

Comment:
I looked at the rest of the file and noticed that `> -1` is the convention in this file as well have one line helper functions.  It's better to go with the flow.